example-config/etesync.ini

```diff
@@ -0,0 +1,18 @@
+# uwsgi configuration file
+# typical location of this file would be /etc/uwsgi/sites/etesync.ini
+
+[uwsgi]
+folder = /path/to/etesync
+
+socket = /run/uwsgi/etesync.sock
+chown-socket = EtesyncUser:www-data
+chmod-socket = 660
+vacuum = true
+
+
+uid = etes
+chdir = %(folder)
+home = %(folder)/.venv
+module = etesync_server.wsgi:application
+master = true
+processes = 5
```

Comment:
The load on a single-user server is minimal and no need for this level of concurrency.